pootle/apps/pootle_app/management/commands/contributors.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from collections import Counter
+from optparse import make_option
+from django.contrib.auth import get_user_model
+from pootle_app.management.commands import PootleCommand
+from pootle_store.models import Unit
```

Comment:
@jleclanche we have a style guide, followup it.  THe styleguide is very clear that we're extending PEP.  This is just a waste of time for everyone reviewing or commenting in that they have to read this again and again from you.  Follow the project style guide or the PR will hang.  You yourself have written countless times about wanting people to repsect your own projects styleguide, how about taking your own advice here.  You have an option of PR'ing a change to the style guide, but you seem to not want to choose that option but instead drag this up whenever anyone has an issue about your import block.  Which doesn't even follow PEP8.  But can you please stop wasting everyones time on this issue, this is so seriously demotivating for anyone having to review your work.